### PR TITLE
fix: respect basePath in @CheckedTemplate to find templates

### PR DESCRIPTION
**Pull request by fbricon**
Still need to :
- handle corner cases, i.e empty paths, several "/"s
- write tests

### CODE REVIEW

**Comment by angelozerr** [comment]
@fbricon I suppose we have the same issue in JDT, right?

**Comment by fbricon** [comment]
see https://github.com/redhat-developer/quarkus-ls/issues/893

File: src/main/java/com/redhat/devtools/intellij/qute/psi/internal/template/datamodel/CheckedTemplateSupport.java [hunk 1]
```diff
@@ -112,15 +113,32 @@ private static boolean isIgnoreFragments(PsiAnnotation checkedTemplateAnnotation
         return ignoreFragment != null ? ignoreFragment.booleanValue() : false;
     }
 
+    public static String getBasePath(PsiAnnotation checkedTemplateAnnotation) {
```
Comment (angelozerr) [inline src/main/java/com/redhat/devtools/intellij/qute/psi/internal/template/datamodel/CheckedTemplateSupport.java:116]: Add Javdoc

**Comment by angelozerr** [comment]
After adding Javadoc, I think we can merge this nice PR.

File: src/main/java/com/redhat/devtools/intellij/qute/psi/internal/template/datamodel/CheckedTemplateSupport.java (hunk 1 above)
Comment (fbricon) [reply src/main/java/com/redhat/devtools/intellij/qute/psi/internal/template/datamodel/CheckedTemplateSupport.java:125]: done

**Comment by angelozerr** [comment]
Thanks @fbricon !
